model/perf.go

```diff
@@ -718,31 +737,32 @@ func makeTimeSeriesPipeline(timeSeriesId TimeSeriesId) []bson.M {
 		},
 		{
 			"$project": bson.M{
-				"measurement": "$rollups.stats.name",
-				"value":       "$rollups.stats.val",
-				"order":       "$info.order",
-			},
-		},
-		{
-			"$group": bson.M{
-				"_id": "$measurement",
-				"time_series": bson.M{
-					"$push": bson.M{
-						"id":    "$_id",
-						"value": "$value",
-						"order": "$order",
-					},
-				},
+				"value": "$rollups.stats.val",
+				"order": "$info.order",
```

Comment:
does anything here populate the _id field? 

are any of the key names stored in variables?

Reply:
I think `_id` carries over automatically in mongodb.

Perhaps, do you mean in like `PerformanceResult` struct?

Reply:
exactly using bsonutil.MustHaveTag()